lecli/apiutils.py

```diff
@@ -1,26 +1,34 @@
 import ConfigParser
 import base64
-import datetime
 import hashlib
 import hmac
+import os
+
+import datetime
+from appdirs import user_config_dir
 
+CONFIG = ConfigParser.ConfigParser()
 
-configfile = 'config.ini'
-config = ConfigParser.ConfigParser()
 
 def load_config():
-  files_read = config.read(configfile)
```

Comment:
I'm inclined to think this should live in the application module (lecli.py)
Is config an api utility or is it application config?